packages/shared/web/src/header/header.tsx

```diff
@@ -60,6 +63,19 @@ export function Header({ children }: HeaderProps) {
     setInLocalStorage('hideSubGraphError', true);
   };
 
+  useEffect(() => {
```

Comment:
Should this be done in the App.tsx file instead?

Reply:
Imo we keep this in the header. We want the local state update happening in the header component. It guarantees that the mobile nav is applied immediately.